boards/arm/google_kukui/pinmux.c

```diff
@@ -13,14 +13,6 @@
 #include <pinmux/stm32/pinmux_stm32.h>
 
 static const struct pin_config pinconf[] = {
```

Comment:
Do we need to keep this file now that `pinconf[]` is empty?

Reply:
I chose to keep it in order not to block board peripheral additions during the transition.
Of course we should avoid addition of peripherals that should be handled in dts.

Reply:
OK to remove this file from CODEOWNERS perspective on this one :)

Will let you make the call @erwango